programs/ui-wrapper/src/instruction.rs

```diff
@@ -0,0 +1,74 @@
+use num_enum::TryFromPrimitive;
+use shank::ShankInstruction;
+
+/// Instructions available for the Manifest ui-wrapper program
+#[repr(u8)]
+#[derive(TryFromPrimitive, Debug, Copy, Clone, ShankInstruction, PartialEq, Eq)]
+#[rustfmt::skip]
+pub enum ManifestWrapperInstruction {
+    // Create a market is not needed for the wrapper
+
+    /// Create a wrapper for owner. Note that owner and payer are separate for
+    /// use as a PDA.
+    #[account(0, writable, signer, name = "owner", desc = "Owner of the Manifest account")]
+    #[account(1, name = "system_program", desc = "System program")]
+    #[account(2, writable, signer, name = "payer", desc = "Payer of rent and gas")]
+    #[account(3, writable, name = "wrapper_state", desc = "Wrapper state")]
+    CreateWrapper= 0,
+
+    /// Allocate a seat. Also initializes this wrapper state
+    #[account(0, name = "manifest_program", desc = "Manifest program")]
+    #[account(1, writable, signer, name = "owner", desc = "Owner of the Manifest account")]
+    #[account(2, writable, name = "market", desc = "Account holding all market state")]
+    #[account(3, name = "system_program", desc = "System program")]
+    #[account(4, writable, signer, name = "payer", desc = "Payer of rent and gas")]
+    #[account(5, writable, name = "wrapper_state", desc = "Wrapper state")]
+    ClaimSeat = 1,
+
+    /// Place order, deposits additional funds needed
+    #[account(0, name = "manifest_program", desc = "Manifest program")]
+    #[account(1, writable, signer, name = "owner", desc = "Owner of the Manifest account")]
```

Comment:
is the order based on performance? I like as much the same across ix as possible.
so here would be
manifest program
owner
system program
payer
...

Reply:
you mean across same ix in wrapper vs core or across different instructions in wrapper?

Reply:
across different ix.